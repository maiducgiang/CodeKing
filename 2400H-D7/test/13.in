10
58531 5000
58713 5000
82854 5000
63345 5000
45629 5000
92399 5000
80833 5000
26249 5000
12583 5000
49655 5000
