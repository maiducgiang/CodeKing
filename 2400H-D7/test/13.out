172104770
224601253
705167278
598379193
232258562
426508178
820174417
168049094
121316195
464693947
